swabbie-orca/src/main/kotlin/com/netflix/spinnaker/swabbie/orca/tagging/OrcaTaggingService.kt

```diff
@@ -44,7 +44,8 @@ class OrcaTaggingService(
               OrcaJob(
                 type = tagRequest.type,
                 context = mutableMapOf(
-                  "tags" to listOf(tagRequest.id)
+                  "tags" to listOf(tagRequest.id),
+                  "id" to tagRequest.id
```

Comment:
So `id` needs to be the entity id, and `tags` is a list of strings of the tag id. These shouldn't be the same value I think?

Reply:
fixed it to pass the correct tag and id , tested it locally and looks good now